value/processor/pom.xml

```diff
@@ -169,7 +168,33 @@
                  META-INF/services entry for a class that the compiler has not yet generated. -->
             <arg>-AallowedMissingSerializableExtensionClasses=.*TestStringSerializerFactory</arg>
           </compilerArgs>
+          <annotationProcessorPaths>
+            <path>
+              <groupId>com.google.auto.service</groupId>
+              <artifactId>auto-service</artifactId>
+              <version>${auto-service.version}</version>
+            </path>
+          </annotationProcessorPaths>
         </configuration>
+        <executions>
+          <execution>
+            <id>default-testCompile</id>
+            <configuration>
+              <annotationProcessorPaths>
+                <path>
+                  <groupId>com.google.auto.value</groupId>
+                  <artifactId>auto-value</artifactId>
+                  <version>HEAD-SNAPSHOT</version>
```

Comment:
That could (should?) probably be `${project.version}`

Reply:
Thanks, yes, that seems better.